physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -715,6 +730,13 @@
   type = real
   kind = kind_phys
   intent = inout
+[fire_type_out]
+  standard_name = fire_type_out
+  long_name = type of fire
+  units = none
```

Comment:
unit should be 1. See https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst

Reply:
@grantfirl Thanks. the unit will be updated to 1.